src/components/card/Card.jsx

```diff
@@ -0,0 +1,29 @@
+import "./Card.css";
+import noImg from "../../imgs/noImg.svg";
+import { getAgoString } from "../../utils/convertTime";
+
+function Card({ description, imageSource, createdAt, url }) {
+  const handleCardClick = () => {
+    window.open(url, "_blank"); // URL을 새 창으로 열기
+  };
+
+  const formattedDate = new Date(createdAt).toLocaleDateString("ko-KR");
+
+  // createdAt 데이터를 기준으로 ago 문자열을 생성
+  const agoString = getAgoString(createdAt);
```

Comment:
👍🏻 👍🏻  너무 좋습니다.

이런 방식도 있습니다. props에서 받아온 값을 가공하여 UI에 바인딩 하는 경우가 많을 때에는 아래 방식을 생각하는 것도 좋은 방식입니다.

**받아오자마자 가공을 하는 방식**
이 접근 방식은 코드의 흐름을 명확히 하고, 가공(`computed` 된 값) 로직이 분산되는 것을 방지하여 코드의 가독성과 유지보수성을 향상시킬 수 있습니다

```suggestion
function Card(props) {
  const { description, imageSource, createdAt, url } = {
     ...props,
     get formattedDate(){
       return new Date(props.createdAt).toLocaleDateString("ko-KR");
     },
     get agoString() {
       return getAgoString(createdAt)
     }
  }
  
  const handleCardClick = () => {
    window.open(url, "_blank"); // URL을 새 창으로 열기
  };
```